TutorLizard.BusinessLogic/Models/User.cs

```diff
@@ -1,20 +1,44 @@
-﻿namespace TutorLizard.BusinessLogic.Models;
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace TutorLizard.BusinessLogic.Models;
 
 public class User
 {
     public int Id { get; set; }
+
+    [Required(ErrorMessage = "Podaj nazwę użytkownika.")]
+    [Display(Name = "Nazwa użytkownika")]
+    [MinLength(5, ErrorMessage = "Minimalna długość: 5 znaków")]
+    [MaxLength(20, ErrorMessage = "Maksymalna długość: 20 znaków")]
     public string Name { get; set; }
+
+    [Display(Name="Typ użytkownika")]
     public UserType UserType { get; set; }
 
-    public User(int id, string name, UserType type)
+    [Required(ErrorMessage = "Podaj adres email.")]
+    [EmailAddress(ErrorMessage = "Podaj prawidłowy adres email.")]
+    [Display(Name = "Adres email")]
+    public string Email { get; set; }
```

Comment:
I'd add maxlength attribute just for safety

Reply:
I missed this comment before merging (I saw green and though it meant "Go!"). Should I put it on my fix branch, or leave it for later? I think we'll be adjusting attributes on all models anyway.

Reply:
it was just a minor nitpick :) We can do this whenever it suits you